server.py

```diff
@@ -1,41 +1,77 @@
 import subprocess
 
 from mcp.server.fastmcp import FastMCP
+from mcp.types import ToolAnnotations
 
 mcp = FastMCP("My App")
 
-@mcp.resource("config://app")
-def get_config() -> str:
-    """Static configuration data"""
-    return "App configuration here"
 
 @mcp.prompt()
-def review_code(code: str) -> str:
-    return f"Please review this code:\n\n{code}"
+def explain_code(code: str, level: str = "beginner") -> str:
+    """Explain code at different levels of detail"""
+    levels = {
+        "beginner": "Please explain this code in simple terms for someone new to programming",
+        "intermediate": "Please explain this code with moderate technical detail",
+        "advanced": "Please provide a detailed technical analysis of this code",
+    }
+    explanation_request = levels.get(level, levels["beginner"])
```

Comment:
Given that level defaults to "beginner" in the function args, I dont think the default value for the .get() is ever needed?